CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ straightforward as possible.
 ### Changed
 - UI: audio button moved into paperclip menu
 - `Chat.Db.BackupDbSupervisor` now receives the name of the DB it needs to supervise as an argument
+- Continuous backup
```

Comment:
merge with `main` please
cause this line is likely to be merged into previous release